eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementClientGenerator.cs

```diff
@@ -60,6 +60,22 @@ protected override void Configure()
             AddVisitor(new TypeFilterVisitor());
             AddVisitor(new PaginationVisitor());
             AddVisitor(new ModelFactoryVisitor());
+            if (IsWirePathEnabled())
```

Comment:
consider to put this into `ManagementInputLibrary`

Reply:
`AddVisitor` should be here for sure, but `IsWirePathEnabled` and its implementation seems closer to `ManagementInputLibrary`